### ISSUE TITLE
export a workload log

### ISSUE THREAD

**Issue by dayyoung0324** [brianfrankcooper/YCSB#1143]
I am a new user of YCSB. I just learned that YCSB can run some predefined workloads. Is it possible to export a workload log? In other words, it is expected that the workload log can include some features for each DB access record, such as 
1. operation: read, insert, or update
2. key: a key id
3. value: the value of a key
4. timestamp: when to execute the operation.

Is that possible for YCSB?

Thank you.

**Comment by busbey** [comment]
I don't believe we output a log of each individual operation as we run against a datastore. You could maybe leverage our htrace integration and have it log to a file, but I'm not sure the trace ID includes the key
